server/rule_engine/ActionPublisher.ts

```
@@ -390,10 +421,12 @@ class ActionPublisher {
 
               return true;
             case ActionType.ENQUEUE_TO_MRT:
-              const fullItemForMrt = await getFullItem();
+              const fullItemForMrt = await getFullItemOrSyntheticUser();
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)